scripts/cat/sprites.py

```diff
@@ -1,22 +1,26 @@
+import logging
```

Comment:
what's up with the logging inclusions?

Reply:
I'm not inventing logging in the other PR, it's already a Python library. It just makes more sense to me to use logging correctly here rather than have to come back here after that PR is merged to change this print to the right statement.